src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityQuantumTank.java

```diff
@@ -134,9 +135,17 @@ public void update() {
                     updatePreviousFluid(null);
                 } else if (previousFluid.getFluid().equals(currentFluid.getFluid()) &&
                         previousFluid.amount != currentFluid.amount) {
+                            int currentFill = MathHelper
+                                    .floor(16 * ((float) currentFluid.amount) / fluidTank.getCapacity());
+                            int previousFill = MathHelper
+                                    .floor(16 * ((float) previousFluid.amount) / fluidTank.getCapacity());
                             // tank has fluid with changed amount
                             previousFluid.amount = currentFluid.amount;
-                            writeCustomData(UPDATE_FLUID_AMOUNT, buf -> buf.writeInt(currentFluid.amount));
+                            writeCustomData(UPDATE_FLUID_AMOUNT, buf -> {
+                                buf.writeInt(currentFluid.amount);
+                                buf.writeBoolean(currentFill != previousFill);
```

Comment:
This doesn't seem to be used at all to space out rendering updates when reading the custom data

Reply:
hmm, i might have messed something up when i was rebasing this